python/eos/analysis_file_description.py

```diff
@@ -38,7 +38,7 @@ class CurtailedGaussianDescription(Deserializable):
     sigma:float
     min:float
     max:float
-    type:str=field(repr=False, init=False, default="curtailed gaussian")
+    type:str=field(repr=False, init=False, default="gaussian")
```

Comment:
This is weird. Why isn't this fix in ``master`` yet?

Reply:
It is in master, your branch isn't properly up to date